build-hypre/action.yml

```diff
@@ -30,25 +30,43 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Get Hypre (Windows)
+      if: runner.os == 'Windows'
+      run: |
+        $URI_HYPRE = "${{ inputs.url }}/${{ inputs.archive }}"
+        Invoke-WebRequest -Uri $URI_HYPRE -OutFile ${{ inputs.archive }}
+        Remove-Item ${{ inputs.dir }} -Force -Recurse -ErrorAction Ignore;
+        tar -xzf ${{ inputs.archive }};
+      shell: pwsh
+
+    - name: Get Hypre (Linux/macOS)
+      if: runner.os != 'Windows'
+      run: |
+        wget --no-verbose ${{ inputs.url }}/${{ inputs.archive }};
+        rm -rf ${{ inputs.dir }};
+        tar -xzf ${{ inputs.archive }};
+      shell: bash
+
     - name: Install Hypre
       run: |
         echo "Map target to options"
         if [[ "${{ inputs.target }}" == "int32" ]]
         then
-          export hypre_options="--disable-fortran";
```

Comment:
Why is this removed? 

(its purpose is to save a bit of build time as we don't need hypre's Fortran dependencies)

Reply:
When I looked into the Hypre documentation some time ago I couldn't find any reference to this option. Are you sure it is still a thing? We had a brief chat with @v-dobrev about it, and I believe he was also thinking that maybe we should remove it.

Reply:
It is indeed been many years since this was added, but I still see it here: https://github.com/hypre-space/hypre/blob/master/src/configure#L1514

Maybe it is not doing anything anymore? Part of the reason to use it was that otherwise one needed a Fortran compiler, which may not be available by default e.g. on a Mac. 

If that's not an issue anymore, I am OK with removing it.